addons/web/static/src/webclient/user_menu/user_menu.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="web.UserMenu" owl="1">
         <Dropdown class="o_user_menu" t-on-dropdown-item-selected.stop="onDropdownItemSelected">
             <t t-set-slot="toggler">
-                <img class="rounded-circle o_user_avatar" t-att-src="source" alt="User"/> <span class="oe_topbar_name"><t t-esc="user.name"/><t t-if="env.debug" t-esc="' (' + user.db.name + ')'"/></span>
+                <img class="rounded-circle o_user_avatar" t-att-src="source" alt="User"/> <span class="oe_topbar_name text-truncate" t-att-title="user.name"><t t-esc="user.name"/><t t-if="env.debug" t-esc="' (' + user.db.name + ')'"/></span>
```

Comment:
Hello @odoo/rd-framework-js ,

Can we have a review for that part? 

Thanks in advance! 🙂

Reply:
Seems good but the DB name is missing in debug mode in the title. Can to you add it. Thx

Reply:
Fp don't like title because it's always displayed even if the text is not truncated... As this screen is often used, I would like to avoid it.

Reply:
So do I need to remove title or add db name in title? :thinking: 
I also think there should be no title as having long title to trigger truncate is quite rare

What do you think?

Reply:
I have done the necessary changes. 
Thank You :)